packages/types/src/clerk.ts

```diff
@@ -541,6 +541,20 @@ export interface Resources {
 
 export type RoutingStrategy = 'path' | 'hash' | 'virtual';
 
+export type SignInInitialValues = {
+  emailAddress?: string;
+  phoneNumber?: string;
+  username?: string;
+};
+
+export type SignUpInitialValues = {
+  emailAddress?: string;
+  phoneNumber?: string;
+  firstName?: string;
+  lastName?: string;
+  username?: string;
+};
```

Comment:
🙃
```suggestion
export type SignUpInitialValues = SignInInitialValues & {
  firstName?: string;
  lastName?: string;
};
```

Reply:
I prefer them separated. They don't necessarily have to overlap.